mod/groups/actions/groups/edit.php

```diff
@@ -94,7 +94,7 @@
 }
 
 // Set group tool options
-$tool_options = elgg_get_config('group_tool_options');
+$tool_options = groups_get_group_tool_options($group);
```

Comment:
Either way works. I'm OK with `groups_get_group_tool_options` *requiring* an ElggGroup as long as we warn hook users to verify `$var['entity']->guid` and not to save().